jest.config.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
     '<rootDir>/src/utils/test-deprecations.tsx',
     '<rootDir>/src/utils/test-helpers.tsx'
   ],
-  testMatch: ['<rootDir>/(src|codemods)/__tests__/**/*.test.[jt]s?(x)', '!**/*.types.test.[jt]s?(x)'],
+  testMatch: ['<rootDir>/(src|codemods)/**/*.test.[jt]s?(x)', '!**/*.types.test.[jt]s?(x)'],
```

Comment:
ooooh collocation 👍

Reply:
We might want to exclude stories from showing up in coverage with [`collectCoverageFrom`](https://jestjs.io/docs/configuration#collectcoveragefrom-array)

<img width="830" alt="image" src="https://user-images.githubusercontent.com/1863771/154483772-81fc2671-8a25-4773-9b4b-aefc3da9c14d.png">